scripts/prepack.ts

```diff
@@ -10,16 +10,24 @@ import * as fs from 'fs';
 import * as fse from 'fs-extra';
 import * as path from 'path';
 
-const NPM_BUILD_DIR = 'build/npm';
 const BUILD_DIR = 'build';
-const NPM_IGNORE = fs.existsSync('.npmignore') ? '.npmignore' : '../../.npmignore';
```

Comment:
This could throw and we're not capturing it like the rest of our `fs` calls. Should we change it for consistency and better devEx - wdyt?

Reply:
Oh I'm just realizing now that this was in fact changed in the previous PR. Sorry, didn't mean to add it to yours. Nevertheless, it's probably something we could address here if it's ok for you?

Reply:
I'm gonna leave it like this. If this crashes, it crashes and thats probably fine and easy to debug anyways.